rest-servlet/src/main/java/org/hawkular/inventory/rest/Security.java

```diff
@@ -152,7 +153,7 @@ public static String getStableId(Class<? extends AbstractElement<?, ?>> type, St
                 return join(ids[0], ids[1], ids[2], "metrics", ids[3]);
             }
         } else if (Relationship.class.isAssignableFrom(type)) {
-            return join("relationships", ids[0]);
+            return "relationships/" + ids[0];
```

Comment:
this is here because of this https://github.com/Jiri-Kremser/hawkular-inventory/blob/a0d8919e2d9715654d5041ced2daac96a6ce99a8/rest-servlet/src/main/java/org/hawkular/inventory/rest/Security.java#L165